vsdkx/addon/tracking/processor.py

```diff
@@ -49,15 +65,10 @@ def post_process(self, addon_object: AddonObject) -> AddonObject:
         # centroids of bounding boxes
         last_updated = {}
 
+        objects, bounding_boxes = self._ct.update(boxes)
         # exit immediately if no people boxes found
-        if len(addon_object.inference.boxes) == 0:
-            self._ct.update(addon_object.inference.boxes)
-            addon_object.inference.extra["tracked_objects"] = event_counter
```

Comment:
@gujjs 'tracked_objects' is the amount of tracked objects and 'trackable_object' are the objects that were tracked on the current frame.

Reply:
I don't get it. is something wrong here?

Reply:
no just ignore it, it's connected to the previous comment about renaming these values that's associated to another task.